### PR TITLE
Help: Localizing inline contextual help links

### PR DESCRIPTION
**Pull request by ramonjd**
## Background 
This is a WIP PR to localize inline contextual help links, whose text and links are all in English.

screen shot 2018-09-13 at 11 07 37 am

It's a starter task, and part of an @Automattic/i18n effort to improve translations across our support sites, a recurring point of pain for our non-English speaking users. 

A subset of this effort is to normalize all support page uris so that https://es.support.wordpress.com**/business-plan/** https://bs.support.wordpress.com**/business-plan/** and https://fr.support.wordpress.com**/business-plan/** redirect to the corresponding, localized support pages.

## What this PR does
When a user's locale matches one for which we have a dedicated support blog and the user clicks an inline contextual help link, we prevent the inline content window (which is currently setup to [fetch English content using post ids](https://github.com/Automattic/wp-calypso/blob/cff5cee/client/blocks/inline-help/contextual-help.js)) from showing and redirect the user to that blog using the English slug.

If we don't yet offer a support blog in the user's language, we use the default `en` functionality.

sep-13-2018 15-34-23

## Future iterations
1. For non-`en` locales where this is no support site, pass a parameter flag `auto_translate=1` to `en.support.wordpress.com` allowing the user to have the support page automatically translated via Google translate .
2. Load localized support page content in the inline content window. This would require getting the post ids of the translated pages and passing them to [QueryReaderPost](https://github.com/Automattic/wp-calypso/blob/cff5cee/client/blocks/support-article-dialog/index.jsx#L72). `D18028-code` is a experimental API endpoint that fetches this information. though its future is uncertain.

## Testing
1. Set your WordPress account language to `es` (or another [support-ready language](https://github.com/Automattic/wp-calypso/blob/08319a51ba68f33798f216ebaf665fdfcdc99852/config/_shared.json#L39))
2. Go to `/stats/`
3. Click on the help icon at the bottom right of the screen, and select one of the contextual help links.
4. Do the same for `ar` or another locale without a WordPress support site.

#### Expectations
**At 3**: You should be taken to the page at `es.support.wordpress.com`. If the localized content does not exist you'll be offered to view the page in English or have it translated using Google Translate

**At 4**: The content will load in English.

It should also work for locale variants such as `de_formal`, which will default to the root language, in this case `de.support.wordpress.com`

### CODE REVIEW

File: client/blocks/inline-help/contextual-help.js [hunk 1]
```diff
@@ -16,34 +17,40 @@ const fallbackLinks = [
...
+			'Set up your domain whether it’s registered with WordPress.com or elsewhere.'
+		),
 	},
 	{
 		link: 'https://en.support.wordpress.com/start/',
 		post_id: 81083,
-		title: 'Quick-Start Guide',
-		description:
-			"Use our five-step checklist to get set up and ready to publish, no matter what kind of site you're building.",
+		title: translate( 'Quick-Start Guide' ),
```
Comment (a8ci18n) [inline client/blocks/inline-help/contextual-help.js]: Hi! I've found a possible matching string that has already been translated **5** times:
`translate( 'Quick Start Guide' )` *ES Score: 15*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).

File: client/blocks/inline-help/contextual-help.js [hunk 2]
```diff
@@ -52,700 +59,795 @@ const contextLinksForSection = {
...
+			description: translate(
+				'Want more traffic? Here are some tips for attracting more visitors to your site.'
+			),
 		},
 		{
 			link: 'https://en.support.wordpress.com/increase-your-site-traffic/',
 			post_id: 132186,
-			title: 'Increase Your Site Traffic',
-			description:
+			title: translate( 'Increase Your Site Traffic' ),
```
Comment (a8ci18n) [inline client/blocks/inline-help/contextual-help.js]: Hi! I've found a possible matching string that has already been translated **20** times:
`translate( 'Increase Your Traffic' )` *ES Score: 10*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).

File: client/blocks/inline-help/contextual-help.js [hunk 3]
```diff
@@ -52,700 +59,795 @@ const contextLinksForSection = {
...
-				'name when you publish content or comment on WordPress.com sites.',
+					'name when you publish content or comment on WordPress.com sites.'
+			),
 		},
 		{
 			link: 'http://en.support.wordpress.com/gravatars/',
 			post_id: 1338,
-			title: 'Your Profile Picture',
-			description:
+			title: translate( 'Your Profile Picture' ),
```
Comment (a8ci18n) [inline client/blocks/inline-help/contextual-help.js]: Hi! I've found a possible matching string that has already been translated **19** times:
`translate( 'Upload your profile picture' )` *ES Score: 14*
See 2 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/add/localized-support-urls-inline-contextual-help)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).

File: client/blocks/inline-help/contextual-help.js [hunk 4]
```diff
@@ -52,700 +59,795 @@ const contextLinksForSection = {
...
-				'powers the user avatars on WordPress.com.',
+					'powers the user avatars on WordPress.com.'
+			),
 		},
 		{
 			link: 'http://en.support.wordpress.com/account-deactivation/',
 			post_id: 138080,
-			title: 'Account Closure',
-			description: 'Need a fresh start? Learn how to close your account.',
+			title: translate( 'Account Closure' ),
```
Comment (a8ci18n) [inline client/blocks/inline-help/contextual-help.js]: Hi! I've found a possible matching string that has already been translated **18** times:
`translate( 'Confirm account closure' )` *ES Score: 13*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).

File: client/blocks/inline-help/contextual-help.js [hunk 5]
```diff
@@ -52,700 +59,795 @@ const contextLinksForSection = {
...
+					'on a yearly subscription that renews automatically.'
+			),
 		},
 		{
 			link: 'http://en.support.wordpress.com/discover-the-wordpress-com-plans/',
 			post_id: 140323,
-			title: 'Explore the WordPress.com Plans',
-			description:
-				"Upgrading your plan unlocks a ton of features! We'll help you pick the best fit for your needs and goals.",
+			title: translate( 'Explore the WordPress.com Plans' ),
```
Comment (a8ci18n) [inline client/blocks/inline-help/contextual-help.js]: Hi! I've found a possible matching string that has already been translated **8** times:
`translate( 'Explore WordPress.com Plans' )` *ES Score: 12*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).

File: client/blocks/inline-help/contextual-help.js [hunk 6]
```diff
@@ -52,700 +59,795 @@ const contextLinksForSection = {
...
 	],
 	media: [
 		{
 			link: 'http://en.support.wordpress.com/media/',
 			post_id: 853,
-			title: 'The Media Library',
-			description:
-				'The Media Library is where you can manage your images, audio, videos, and documents ' +
-				'all in one place.',
+			title: translate( 'The Media Library' ),
```
Comment (a8ci18n) [inline client/blocks/inline-help/contextual-help.js]: :disappointed: 103 existing translations will be lost with this change.

File: client/blocks/inline-help/contextual-help.js [hunk 7]
```diff
@@ -52,700 +59,795 @@ const contextLinksForSection = {
...
+			description: translate(
+				'Invite contributors, followers, and viewers to collaborate with others and grow your audience!'
+			),
 		},
 		{
 			link: 'http://en.support.wordpress.com/followers/',
 			post_id: 5444,
-			title: 'Your Followers',
-			description:
+			title: translate( 'Your Followers' ),
```
Comment (a8ci18n) [inline client/blocks/inline-help/contextual-help.js]: :disappointed: 56 existing translations will be lost with this change.
If the string must change, the following option has already been translated **43** times:
`translate( 'followers' )` *ES Score: 10*
See 1 additional suggestion in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/add/localized-support-urls-inline-contextual-help)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).

File: client/blocks/inline-help/contextual-help.js [hunk 8]
```diff
@@ -52,700 +59,795 @@ const contextLinksForSection = {
...
+				'The Related Posts feature pulls relevant content from your blog to display at the bottom of your posts.'
+			),
 		},
 		{
 			link: 'http://en.support.wordpress.com/webmaster-tools/',
 			post_id: 5022,
-			title: 'Webmaster Tools',
-			description:
-				'Learn how to verify your WordPress.com site for the webmaster tools that many search engines provide.',
+			title: translate( 'Webmaster Tools' ),
```
Comment (a8ci18n) [inline client/blocks/inline-help/contextual-help.js]: Hi! I've found a possible matching string that has already been translated **17** times:
`translate( 'Webmaster Tools Verification' )` *ES Score: 11*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).

File: client/blocks/inline-help/contextual-help.js [hunk 9]
```diff
@@ -52,700 +59,795 @@ const contextLinksForSection = {
...
+				"Upgrading your plan unlocks a ton of features! We'll help you pick the best fit for your needs and goals."
+			),
 		},
 		{
 			link: 'http://en.support.wordpress.com/plan-features/',
 			post_id: 134698,
-			title: 'WordPress.com Plans',
-			description:
-				'Learn about the capabilities and features that the different plans unlock for your site.',
+			title: translate( 'WordPress.com Plans' ),
```
Comment (a8ci18n) [inline client/blocks/inline-help/contextual-help.js]: Hi! I've found a possible matching string that has already been translated **18** times:
`translate( 'Discover WordPress.com Plans' )` *ES Score: 11*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).

File: client/blocks/inline-help/contextual-help.js [hunk 10]
```diff
@@ -52,700 +59,795 @@ const contextLinksForSection = {
...
-				'available to you: the Visual Editor and the HTML Editor.',
+					'available to you: the Visual Editor and the HTML Editor.'
+			),
 		},
 		{
 			link: 'http://en.support.wordpress.com/visual-editor/',
 			post_id: 3644,
-			title: 'The Visual Editor',
-			description:
+			title: translate( 'The Visual Editor' ),
```
Comment (a8ci18n) [inline client/blocks/inline-help/contextual-help.js]: :disappointed: 95 existing translations will be lost with this change.

File: client/blocks/inline-help/contextual-help.js [hunk 11]
```diff
@@ -52,700 +59,795 @@ const contextLinksForSection = {
...
-				'Discover great new reads and keep track of your comments and replies in one convenient place.',
+					'Discover great new reads and keep track of your comments and replies in one convenient place.'
+			),
 		},
 		{
 			link: 'http://en.support.wordpress.com/following/',
-			title: 'Follow Blogs',
-			description:
-				'When you follow a blog on WordPress.com, new posts from that site will automatically appear in your Reader.',
+			title: translate( 'Follow Blogs' ),
```
Comment (a8ci18n) [inline client/blocks/inline-help/contextual-help.js]: Hi! I've found a possible matching string that has already been translated **59** times:
`translate( 'Blogs I Follow' )` *ES Score: 12*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).

File: client/blocks/inline-help/contextual-help.js [hunk 12]
```diff
@@ -52,700 +59,795 @@ const contextLinksForSection = {
...
+			),
 		},
 	],
 	help: [
 		{
 			link: 'http://en.support.wordpress.com/blogging-u/',
 			post_id: 117437,
-			title: 'Blogging U.',
-			description:
+			title: translate( 'Blogging U.' ),
```
Comment (a8ci18n) [inline client/blocks/inline-help/contextual-help.js]: :ok: This change is safe, we'll keep the existing 21 translations.

File: client/blocks/inline-help/contextual-help.js [hunk 13]
```diff
@@ -52,700 +59,795 @@ const contextLinksForSection = {
...
+			title: translate( 'Help! Getting WordPress.com Support' ),
+			description: translate(
+				'WordPress.com offers a number of avenues for reaching helpful, individualized support.'
+			),
 		},
 		{
 			link: 'https://en.support.wordpress.com/',
-			title: 'All Support Articles',
-			description: 'Looking to learn more about a feature? Our docs have all the details.',
+			title: translate( 'All Support Articles' ),
```
Comment (a8ci18n) [inline client/blocks/inline-help/contextual-help.js]: Hi! I've found a possible matching string that has already been translated **34** times:
`translate( 'All support articles' )` *ES Score: 16*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).

File: client/blocks/inline-help/contextual-help.js [hunk 14]
```diff
@@ -52,700 +59,795 @@ const contextLinksForSection = {
...
+			description: translate(
+				'Looking to learn more about a feature? Our docs have all the details.'
+			),
 		},
 		{
 			link: 'https://learn.wordpress.com/',
-			title: 'Self-guided Online Tutorial',
-			description: 'A step-by-step guide to getting familiar with the platform.',
+			title: translate( 'Self-guided Online Tutorial' ),
```
Comment (a8ci18n) [inline client/blocks/inline-help/contextual-help.js]: Hi! I've found a possible matching string that has already been translated **25** times:
`translate( 'Self-guided online tutorial' )` *ES Score: 23*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).

File: client/blocks/inline-help/contextual-help.js [hunk 15]
```diff
@@ -766,329 +868,371 @@ const videosForSection = {
...
-				'do on your Jetpack-enabled WordPress site.',
+					'do on your Jetpack-enabled WordPress site.'
+			),
 		},
 		{
 			type: RESULT_VIDEO,
 			link: 'https://www.youtube.com/embed/Y6iPsPwYD7g',
-			title: 'Change Your Privacy Settings',
-			description: 'Find out how to change your website privacy settings on WordPress.com.',
+			title: translate( 'Change Your Privacy Settings' ),
```
Comment (a8ci18n) [inline client/blocks/inline-help/contextual-help.js]: Hi! I've found a possible matching string that has already been translated **24** times:
`translate( 'Change privacy settings' )` *ES Score: 12*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).

File: client/blocks/inline-help/contextual-help.js [hunk 16]
```diff
@@ -766,329 +868,371 @@ const videosForSection = {
...
-				'on your WordPress.com or Jetpack-enabled WordPress website or blog.',
+					'on your WordPress.com or Jetpack-enabled WordPress website or blog.'
+			),
 		},
 		{
 			type: RESULT_VIDEO,
 			link: 'https://www.youtube.com/embed/_tpcHN6ZtKM',
-			title: 'Schedule a Post',
-			description: 'Find out how to schedule a post on your WordPress.com website or blog.',
+			title: translate( 'Schedule a Post' ),
```
Comment (a8ci18n) [inline client/blocks/inline-help/contextual-help.js]: Hi! I've found a possible matching string that has already been translated **21** times:
`translate( 'Schedule post' )` *ES Score: 12*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).

File: client/blocks/inline-help/contextual-help.js [hunk 17]
```diff
@@ -766,329 +868,371 @@ const videosForSection = {
...
+			description: translate(
+				'Find out how to schedule a post on your WordPress.com website or blog.'
+			),
 		},
 		{
 			type: RESULT_VIDEO,
 			link: 'https://www.youtube.com/embed/V8UToJoSf4Q',
-			title: 'Add a Simple Payment Button',
-			description: 'Find out how to add a payment button to your WordPress.com website.',
+			title: translate( 'Add a Simple Payment Button' ),
```
Comment (a8ci18n) [inline client/blocks/inline-help/contextual-help.js]: Hi! I've found a possible matching string that has already been translated **1** time:
`translate( 'Add a simple payment button.' )` *ES Score: 17*
See 1 additional suggestion in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/add/localized-support-urls-inline-contextual-help)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).

File: client/blocks/inline-help/contextual-help.js [hunk 18]
```diff
@@ -766,329 +868,371 @@ const videosForSection = {
...
-			description: 'Find out how to create a page on your WordPress.com site.',
+			title: translate( 'Create a Page' ),
+			description: translate( 'Find out how to create a page on your WordPress.com site.' ),
 		},
 		{
 			type: RESULT_VIDEO,
 			link: 'https://www.youtube.com/embed/mCfuh5bCOwM',
-			title: 'Create a Post',
-			description: 'Find out how to create a post on WordPress.com.',
+			title: translate( 'Create a Post' ),
```
Comment (a8ci18n) [inline client/blocks/inline-help/contextual-help.js]: Hi! I've found a possible matching string that has already been translated **26** times:
`translate( 'Create a post' )` *ES Score: 10*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).

File: client/blocks/inline-help/contextual-help.js [hunk 19]
```diff
@@ -766,329 +868,371 @@ const videosForSection = {
...
+			description: translate(
+				'Find out how to show related posts on your WordPress.com site, which you can also do on a Jetpack-enabled WordPress blog.'
+			),
 		},
 		{
 			type: RESULT_VIDEO,
 			link: 'https://www.youtube.com/embed/JVnltCZUKC4',
-			title: 'Add Testimonials',
-			description: 'Find out how to add testimonials to your WordPress.com website or blog.',
+			title: translate( 'Add Testimonials' ),
```
Comment (a8ci18n) [inline client/blocks/inline-help/contextual-help.js]: Hi! I've found a possible matching string that has already been translated **18** times:
`translate( 'Add some testimonials' )` *ES Score: 13*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).

**Comment by lsinger** [comment]
Inline Help uses a file called `contextual-help.js` as its database, which I see you've edited heavily in this PR to add a locale variable to every entry. 

When we built Inline Help, the idea here was that we could, in the future, add *alternative version of the file for different languages*, e.g., a file called `contextual-help-br.js` could contain the help content for Brazil. We'd only need to change the import of that file to be based on the user's locale. That would also enable us to include languages that don't have all the listed help articles available, or that maybe have locale-specific issues (legal things, payment services, differing social media sites, etc.). We'd just have a custom "database" for the locale. 

I'd prefer we explore that option before we make the `contextual-help.js` file itself dynamic. Thoughts? Please feel free to ping me on Slack (also: `#calypso-inline-help`!).

**Comment by ramonjd** [comment]
Hey @lsinger! Thanks a lot for your comments. It's immensely helpful to understand the intentions behind this feature.

> When we built Inline Help, the idea here was that we could, in the future, add alternative version of the file for different languages, e.g., a file called contextual-help-br.js could contain the help content for Brazil. We'd only need to change the import of that file to be based on the user's locale

Would the contents of each of these files be translated? Or hardcoded in the target language, i.e., Brazilian? 

If the latter, we'd probably have to come up with a way to generate these from our translation file, which compels me to ask whether we could rather devise some backend solution/API to deliver localized contextual help text and support links to the client. 

Storing them on the client side up to a certain quantity sounds easier, yet we might gain more flexibility later if we could fetch the pageId, title, excerpt and permalink of the localized support article we wanted via, let's say, an English slug or pageId. 

@Automattic/i18n is working on adding meta to localized support pages, linking them to their English equivalents, so this will be theoretically possible.

It strikes me as particularly relevant in the case of the hard-coded page id, which I noticed we use for tracking and for loading the support article's content in the overlay. Keeping on top of those as they change and as we add more languages could become tedious.

What do you think?

> That would also enable us to include languages that don't have all the listed help articles available

@Automattic/i18n is also steaming ahead with getting our support translations up to scratch, at least for our "Magnificent" set of languages. @emilyaudela is doing a marvellous job on that. 

One advantage of the meta information I referred to above is that we'll be able to query whether a localized support page exists for a particular topic, instead of curating a manual list or simply linking off to the English language page, which is the case now.

Our ultimate goal is to standardize the support help links in such a way so that developers only have to worry about the locale subdomain fragment when linking to `{x}.support.wordpress.com`, and not whether the page exists at all. 

For languages that fall beyond the bounds of magnificence, @akirk and I spoke about perhaps offering users the option to have the English page machine translated (Google) via a url param or whatever. That's a few steps down the track however.

> ...maybe have locale-specific issues (legal things, payment services, differing social media sites, etc.) We'd just have a custom "database" for the locale. I'd prefer we explore that option before we make the contextual-help.js file itself dynamic. 

Has any work been down on locale-specific issues? Perhaps in the interim we might offer a generic set of help topics for each locale, in seperate resources as you mentioned, and then extend them with locale-specific topics as we progress. That way at least we could offer contextual help in languages other than English.

> I'd prefer we explore that option before we make the contextual-help.js file itself dynamic. 

Just to be clear, are you referring to the dynamic help links or the use of `translate` in general? 



Ah, I wrote more than I intended. 🥔I hope some of it makes sense. Thanks again!

I'll join the group so I can get more context. Getting this awesome feature translated would be a tremendous thing indeed!

**Comment by ramonjd** [comment]
Could we not start off with something like this PR? Open to alternatives of course, however it would be great to have something on offer in languages other than English, at least until work progresses. We can always roll back changes.

cc @akirk

**Comment by lsinger** [comment]
I’m generally in favor of trying things and running with them until they don’t scale anymore and then iterating on them. Of course, feel free to do so. 

(And that’s also the reason why we use a file as our database. As long as that works without pain we don’t need to scale it up.)

My impression though is that you’re assuming that all help articles will exist in every language. I don’t think that will necessarily hold true, so you’d need to make sure that whatever solution you run with accounts for that.

**Comment by ramonjd** [comment]
> My impression though is that you’re assuming that all help articles will exist in every language.

@lsinger They don't. Guarding against such this assumption has always been an unfortunate limitation on how we link to support sites, but we've done some preliminary work to mitigate this. 

Perhaps I should have provided more background information. 

As mentioned in the PR, we are normalizing all support page uris so that de.support.wordpress.com`/business-plan/`, es.support.wordpress.com`/business-plan/` and fr.support.wordpress.com`/business-plan/` redirect to the corresponding, localized support pages. 

Where they don't, the user will see a localized page offering them the choice of visiting the original English or a Google-translated version. Though as more and more support page translations come through, we should hopefully experience fewer of these.

More info: `pLZE4-cYQ-p2`

If you have any questions on this, or would like to share ideas, feel free to ping me directly.

**Comment by akirk** [comment]
Could you incorporate #27715?

File: client/blocks/inline-help/contextual-help.js [hunk 20]
```diff
@@ -766,329 +869,371 @@ const videosForSection = {
...
+			title: translate( 'Change Your WordPress.com Display Name' ),
+			description: translate( 'Find out how to change your display name on WordPress.com.' ),
 		},
 		{
 			type: RESULT_VIDEO,
 			link: 'https://www.youtube.com/embed/07Nf8FkjO4o',
-			title: 'Change Your Account Email Address',
-			description: 'Find out how to change your account email address on WordPress.com.',
+			title: translate( 'Change Your Account Email Address' ),
+			description: translate( 'Find out how to change your account email address on WordPress.com.' ),
```
Comment (a8ci18n) [inline client/blocks/inline-help/contextual-help.js]: Hi! I've found a possible matching string that has already been translated **18** times:
`translate( 'Find out how to change your account email address WordPress.com.' )` *ES Score: 11*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).

File: config/_shared.json [hunk 21]
```diff
@@ -36,7 +36,7 @@
 	"facebook_app_id": "611241942420191",
 	"rebrand_cities_prefix": "rebrandcitiestestsite",
     "livechat_support_locales": [ "en", "es", "pt-br" ],
-    "support_site_locales": [ "en", "es", "de", "ja", "pt-br" ],
+    "support_site_locales": [ "en", "es", "de", "fr", "he", "id", "it", "ja", "nl", "pt-br", "ru", "tr" ],
```
Comment (akirk) [inline config/_shared.json]: We've now launched all Mag16. Could you add those and I think then we can launch this!

**Comment by akirk** [comment]
Would it make sense to rebase this on #27173?

**Comment by ramonjd** [comment]
> Would it make sense to rebase this on #27857?

Yes!! :) When are you going to write a common-sense bot? :)

Actually, given the size/usefulness of #27857 it would be great to get that into master first. I'll review today.

**Comment by akirk** [comment]
In light of #27857 and #28166 I have removed the use of `getSupportSiteLocale()`.

**Comment by ramonjd** [comment]
Rebased and updated to use `localizeUrl()` from #27857

File: client/blocks/inline-help/contextual-help.js [hunk 22]
```diff
@@ -3,749 +3,866 @@
...
-		description:
-			'If you’re building a brand new site, you might be wondering if you need a website, a blog, or a website with a blog. At WordPress.com, you can create all of these options easily, right in your dashboard.',
+		title: translate( 'Do I Need a Website, a Blog, or a Website with a Blog?' ),
+		description: translate(
+			'If you’re building a brand new site, you might be wondering if you need a website, a blog, or a website with a blog. At WordPress.com, you can create all of these options easily, right in your dashboard.'
+		),
 	},
 	{
-		link: 'https://en.support.wordpress.com/business-plan/',
+		link: localizeUrl( localizeUrl( 'https://en.support.wordpress.com/business-plan/' ) ),
```
Comment (akirk) [inline client/blocks/inline-help/contextual-help.js]: Is the double `localizeUrl()` intentional? Actually, we don't need it here as the localizeUrl already happens in https://github.com/Automattic/wp-calypso/pull/28166/files#diff-9d01b2007d686a6ec3241f54108060d8R53

Comment (ramonjd) [reply client/blocks/inline-help/contextual-help.js]: Eeek! Bad regex!

**Pull request merged**